acct-group/icecream/icecream-0.ebuild

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION='Group for sys-devel/icecream'
+
+ACCT_GROUP_ID=397
```

Comment:
I suggest to use -1 on both acct-group/user. This will be fixed by devs on merge